README.md

```diff
@@ -2098,14 +2099,7 @@ set :protection, :except => [:path_traversal, :session_hijacking]
 ```
 
 By default, Sinatra will only set up session based protection if `:sessions`
-has been enabled. Sometimes you want to set up sessions on your own, though. In
-that case you can get it to set up session based protections by passing the
-`:session` option:
-
-```ruby
-use Rack::Session::Pool
-set :protection, :session => true
-```
+has been enabled.
```

Comment:
Sounds good. To enable session based protection using the middleware option the following code would need to be used though (as described in #1038):

``` 
 disable :protection
 use Rack::Session::Pool
 use Rack::Protection
```
or
```
use Rack::Session::Pool
use ::Rack::Protection::RemoteToken
use ::Rack::Protection::SessionHijacking
```
The code sample used before was incorrect.